public/doc/openapi-3-v1.2.json

```diff
@@ -73,8 +73,8 @@
                 "tags": [
                   "Action"
                 ],
-                "summary": "Return an user action by id",
-                "description": "Return an user action by id, available to all",
+                "summary": "Return an action",
```

Comment:
To match with other summary/description patterns, should change to like following?
```
"summary": "Find an action",
"description": "Return an action by its id, available to everyone",
```

Reply:
Throughout the doc always `summary` => 'Return`, `description` => `Find`